tethys_sdk/components/utils.py

```diff
@@ -0,0 +1,2 @@
+from reactpy import component, event as event_decorator  # noqa: F401
```

Comment:
Yeah, we could just leave it as event. In the cases that I used it, I would end up wanting `event` to be an argument of some of the functions used surrounding it. I just didn't want any namespace confusion. But it would make more sense to rename my function args rather than the decorator.